packages/contentful-extension-scripts/scripts/utils/updateExtension.js

```diff
@@ -0,0 +1,12 @@
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
+
+module.exports = async (https = false, port = 1234) => {
+  const { stdout, stderr } = await exec(
```

Comment:
Will it be always resolved? 

I think we need to throw an Error if the command failed.

Reply:
Hmm, interesting point. I'm still doubting of throwing an error will fix anything. When the exec command fails, because it throws an error the promise, the whole function will throw the error that is thrown by the "contentful extension update" function. Of course, I could easily overwrite this behavior by catching the error and throw something else, but I'm afraid that this would make it less clear for users.  

Right now I don't know what kind of error message I can expect from the "contentful extension update" command. So I prefer to throw the errors in the same way as we get it, instead of overwriting it with something else. 

What do you think? If you disagree with it, what kind of error message would you expect?

Reply:
I'm okay with showing exactly the same error that was printed by `contentful extension update`. I just wanted to confirm that `await exec` throws an Error at all because by looking at `{ stdout, stderr }` variables I assumed that it's always be resolved even if the command failed. 

Can you double check this by intentionally breaking the command?

Reply:
I've checked it and found out that the await exec throws the same exception as the contentful extension update command. After some testing I've figured even out what the difference is between the exception and stdout and Stderr:

**stdout:** Are all console.log messages that already are displayed to the user by the Contentful extension update command in one big string. 

**stderr:** Are all console.error messages that already are displayed to the user by the Contentful extension update command in one big string. 

Because all messages are already displayed to the user, it doesn't make any sense to repeat these. So for that reason, I deleted both console.log commands.